config/v1/zz_generated.crd-manifests/0000_10_openshift-controller-manager_01_builds.crd.yaml

```diff
@@ -146,9 +146,15 @@ spec:
                                   description: The key to select.
                                   type: string
                                 name:
-                                  description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
-                                    TODO: Add other useful fields. apiVersion, kind,
-                                    uid?'
+                                  default: ""
+                                  description: 'Name of the referent. This field is
```

Comment:
Where did we get this difference from?

Reply:
Coming from k8s 1.30.1, see the other commit here https://github.com/openshift/api/pull/1906/commits/f4098b528011ac02d84c344c1b3e20f88aeca09f#diff-42206d7b9766bf3f301053e91bdc4d95dbbd501c48bad91d141689177c716b00